fem/linearform.hpp

```diff
@@ -185,10 +209,15 @@ class LinearForm : public Vector
        corresponding pointer (to Array<int>) will be NULL. */
    Array<Array<int>*> *GetFLFI_Marker() { return &boundary_face_integs_marker; }
 
+   /// Which assembly algorithm to use: the new device-compatible fast assembly
+   /// (true), or the legacy CPU-only algorithm (false). If not set, the default
+   /// value is false.  If used, this method must be called before assembly. */
+   void UseFastAssembly(bool use_fa);
+
    /// Assembles the linear form i.e. sums over all domain/bdr integrators.
-   /// When @a use_device is set to true and the linearform assembly is
+   /// When UseFastAssembly(true) has been called and the linearform assembly is
    /// compatible with device execution, it will be executed on the device.
```

Comment:
```c++
   /// Assembles the linear form i.e. sums over all domain/bdr integrators.
   /** When @ref UseFastAssembly "UseFastAssembly(true)" has been called and the
       linearform assembly is compatible with device execution, it will be
       executed on the device. */
```

Reply:
Fixed in https://github.com/mfem/mfem/pull/3235/commits/343e7ca7c8c5f4b432f97f8632702f8ce5ef2c6c